src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx

```diff
@@ -6,29 +6,25 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-As part of New Relic's support for OpenTelemetry Protocol (OTLP) attributes can contain arrays. Arrays are a data structure that contains an ordered collection of values.
+As part of our support for OpenTelemetry Protocol (OTLP), attributes can contain arrays within New Relic. But what are arrays? Essentially, arrays are data structures containing an ordered collection of values that you can query with NRQL. Typically, arrays display in a format like `[1, 2, 3, 4, 5]` or `["US", "CA", "UK"]`.
```

Comment:
Should it be "the OpenTelemetry Protocol"?

Reply:
There's something a little unclear about that first sentence. I'm assuming that OTLP can send data formatted as arrays to New Relic and New Relic can process and represent those arrays AS arrays, instead of munging them some other way. Is that what you're trying to convey? Perhaps there's a clearer way to state that.

Also, super torn about the array definition here, because it's such a basic/foundational programming concept